torch/distributed/distributed_c10d.py

```diff
@@ -1688,7 +1688,7 @@ def isend(tensor: torch.Tensor, dst: int, group: Optional[ProcessGroup] = None,
 
     Args:
         tensor (Tensor): Tensor to send.
-        dst (int): Destination rank.
+        dst (int): Destination rank on global process group (regardless of 'group' argument)
```

Comment:
![Screen Shot 2024-01-29 at 8 46 36 PM](https://github.com/pytorch/pytorch/assets/31054793/6320827a-596a-4dda-9174-3b0b9fbce5b1)
nit: We can use \`\`group\`\` if we want `group` in code type.

Reply:
will use \`\`group\`\` for better looking